cluster/ci/config/submit-queue/submit_queue_openshift_ansible.yaml

```diff
@@ -0,0 +1,201 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: submit-queue-openshift-ansible
+  labels:
+    app: mungegithub-openshift-ansible
+    target-repo: openshift-ansible
+parameters:
+- name: NAME
+  displayName: Application Name
+  description: The name to use for the objects created by this template
+  value: submit-queue-openshift-ansible
+  required: true
+- name: SERVICE_NAME
+  displayName: Service Name
+  description: The name to use for the service that will report the SubmitQueue status
+  value: openshift-ansible-sq-status
+  required: true
+- name: SOURCE_URL
+  displayName: Source URL
+  description: Git source URL to use for building the Submit Queue image
+  value: https://github.com/openshift/test-infra.git
+  required: true
+objects:
+- apiVersion: v1
+  kind: Route
+  metadata:
+    name: "${SERVICE_NAME}"
+    labels:
+      app: mungegithub-openshift-ansible
+      component: "${NAME}"
+      target-repo: openshift-ansible
+  spec:
+    to:
+      kind: Service
+      name: "${SERVICE_NAME}"
+    port:
+      targetPort: status
+    tls:
+      termination: edge
+      insecureEdgeTerminationPolicy: Redirect
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: "${SERVICE_NAME}"
+    labels:
+      app: mungegithub-openshift-ansible
+      component: "${NAME}"
+      target-repo: openshift-ansible
+    annotations:
+      prometheus.io/path: /prometheus
+      prometheus.io/scrape: "true"
+      service.openshift.io/infrastructure: 'true'
+  spec:
+    ports:
+    - name: status
+      protocol: TCP
+      port: 80
+      targetPort: 8080
+    selector:
+      app: mungegithub-openshift-ansible
+      component: "${NAME}"
+      target-repo: openshift-ansible
+    type: ClusterIP
+    sessionAffinity: None
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    name: "${NAME}"
+    labels:
+      app: mungegithub-openshift-ansible
+      component: "${NAME}"
+  spec:
+    replicas: 1
+    revisionHistoryLimit: 2
+    template:
+      metadata:
+        labels:
+          app: mungegithub-openshift-ansible
+          component: "${NAME}"
+          target-repo: openshift-ansible
+      spec:
+        containers:
+        - name: "${NAME}"
+          image: "${NAME}"
+          command:
+          - /mungegithub
+          - --dry-run=false
+          - --alsologtostderr
+          - --stderrthreshold=INFO
+          - --config-path=/etc/munge-config/config
+          ports:
+          - name: status
+            containerPort: 8080
+          resources:
+            requests:
+              cpu: 2000m
```

Comment:
Do we need this if we are not going to have approvals for openshift-ansible?

Reply:
no